packages/jetpack-blocks/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.cache/
+/dist/
```

Comment:
Looks like this is already covered by root `.gitignore`:

https://github.com/Automattic/wp-calypso/blob/2b2ca2b01466b33add52c533cb6383cef7080be2/.gitignore#L69-L70

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/b27473f0a65c04955b3d183bb341295c418d919f